templates/expo-template-default/components/web-badge.tsx

```diff
@@ -0,0 +1,29 @@
+import { Image } from 'expo-image';
+import React from 'react';
+import { useColorScheme } from 'react-native';
+
+import { ThemedText } from './themed-text';
+import { ThemedView } from './themed-view';
+
+import { Spacing } from '@/constants/theme';
+import { version } from '@/package.json';
```

Comment:
Importing `package.json` may be a security concern as it leaks all dependencies to the bundle.